src/date-range-picker/embedded.tsx

```diff
@@ -31,7 +30,7 @@ export function DateRangePickerEmbedded({
   timeInputFormat = 'hh:mm:ss',
   rangeSelectorMode = 'default',
   onChange,
-  timeOffset = getBrowserTimezoneOffset(),
+  timeOffset,
```

Comment:
Would this be considered a breaking change?

Reply:
Why? The default value is still applied but inside the utils.